apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/model/Parameters.java

```diff
@@ -1,6 +1,27 @@
 package com.apollocurrrency.aplwallet.inttest.model;
 
 public enum  Parameters {
+    expirationHeight,
```

Comment:
camel case enum members seems to be the violation of enum naming convention. Since enum members belong to constants - use upper case + underscore delimiter to name them. EXPIRATION_HEIGHT.

Reply:
In this format  API will not accept. This enum will be fixed completely in the future.